sources/packages/backend/apps/api/src/route-controllers/application-change-request/_tests_/e2e/application-change-request.aest.controller.pendingApplicationChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,336 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { ApplicationEditStatus, ApplicationStatus } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import MockDate from "mockdate";
+
+describe("ApplicationChangeRequestAESTController(e2e)-pendingApplicationChangeRequest", () => {
```

Comment:
Same here https://github.com/bcgov/SIMS/pull/4733/files#r2107846384

Reply:
Still not right. the description must be `ApplicationChangeRequestAESTController(e2e)-getApplicationChangeRequests`